hibernate-types-4/src/test/java/com/vladmihalcea/hibernate/type/util/AbstractTest.java

```diff
@@ -430,4 +431,19 @@ public void execute(Connection connection) throws SQLException {
             }
         }
     }
+
+    /**
+     * Configuration singleton may "unsuccessfully" cache old value due random JUnit's test run order,
+     * so we need handle to "reinitialize" it.
+     * @param instance
+     */
+    protected void reinitializeConfigurationSingleton(com.vladmihalcea.hibernate.type.util.Configuration instance) {
```

Comment:
In what situation did you need this? I never required it.

Reply:
Tests are broken on my machine if I won' t reset singleton
https://i.imgur.com/xyP1DRz.png